### ISSUE TITLE
Feature : Track the options chosen in FOMOD install

### ISSUE THREAD

**Issue by CreeperLava** [TanninOne/modorganizer#866]
It would be nice if the "Reinstall" feature would recall the options you chose during a FOMOD install.
Or, maybe a simpler way to do it, just automatically write down the options chosen somewhere. Maybe in the "Notes" tab of the mod ?

**Comment by RPINerd** [comment]
Yeah I end up just opening up notepad when reinstalling things like Valdici Item Sorting or Better Mod Descriptions haha
